README.md

```diff
@@ -6,4 +6,50 @@ Base repository for the [Core Data Structures](https://github.com/GuildCrafts/we
 
 ## Installation and Setup
 
-## Usage and Examples
+```npm i```
+
+```npm test```
+
+## Specifications
+
+_List of specifications (specs) for the completed goal. These are declarative sentences (statements) describing a feature of the final product._
+
+**This is a core goal. The specifications are non-negotiable. To complete this goal, you must complete all specs listed below.**
+
+- [x] Artifact produced is a fork of the [core-data-structures][core-data-structures] repo.
+- [x] Can run all tests with `npm test`.
+- [x] All tests are passing.
+- [x] For each data structure identified above, there exists:
+  - [x] a test file with unit tests for each method and property.
+  - [x] an implementation file with a correct implementation of the data structure.
+
+**Data Structures Completed**
+
+- [x] Stack
+- [x] Set
+- [x] Queue
+- [x] Priority Queue
+- [x] Linked List
+- [x] Hash Table _(mostly complete)_
+- [ ] Doubly-Linked List
+- [ ] Binary Search Tree
+- [ ] Directed Graph
+
+## Quality Rubric
+
+**Well formatted code**
+- [ ] Code uses a linter, which can be invoked with a command (e.g. `npm run lint`). [50 points]
+- [ ] Running the linter on all source code files generates no linting errors. [50 points]
```

Comment:
I believe these are immutable goals(if I am not mistaken). A linter would not be difficult to set up and would help complete your specs.